src/test/bundler_test.ts

```diff
@@ -177,14 +177,36 @@ suite('Bundler', () => {
   });
 
   test('lazy imports are not moved', async () => {
-    assert.equal(
-        dom5.queryAll(
-                await bundle('test/html/imports/lazy-imports.html'),
-                preds.AND(
-                    preds.parentMatches(preds.hasTagName('dom-module')),
-                    matchers.htmlImport))
-            .length,
-        2);
+    const bundler = new Bundler({
+      analyzer:
+          new Analyzer({urlLoader: new FSUrlLoader('test/html/imports')})
+    });
+    const manifest = await bundler.generateManifest(['lazy-imports.html']);
+    const documents = await bundler.bundle(manifest);
+    const matchLazyImportInsideDomModule = preds.AND(
+        preds.parentMatches(preds.hasTagName('dom-module')),
+        matchers.htmlImport);
+
+    // The `lazy-imports.html` file has 3 imports in the head of the
+    // document.  The first is eager and should be moved.  The remaining
+    // two are lazy imports and should not be moved.
+    const entrypointBundle = documents.get('lazy-imports.html')!.ast;
+    console.log(parse5.serialize(entrypointBundle));
+    const entrypointLazyImports = dom5.queryAll(
+        entrypointBundle,
+        preds.AND(preds.parentMatches(matchers.head), matchers.htmlImport));
```

Comment:
In the `<head>` case here, I do want to match all html imports so I can verify the eager one is removed by verifying the number of imports matching is 2 instead of 3.